cpp/pybind/visualization/visualizer.cpp

```diff
@@ -112,9 +112,11 @@ void pybind_visualizer(py::module &m) {
                  "reset_bounding_box"_a = true)
             .def("clear_geometries", &Visualizer::ClearGeometries,
                  "Function to clear geometries from the visualizer")
-            .def("get_view_control", &Visualizer::GetViewControl,
-                 "Function to retrieve the associated ``ViewControl``",
-                 py::return_value_policy::reference_internal)
+            .def(
+                    "get_view_control",
+                    [](Visualizer &vis) { return vis.GetViewControl(); },
```

Comment:
What was the purpose of this change?

Reply:
These changes were reverted in this PR:
https://github.com/isl-org/Open3D/pull/6116/files